src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search search--big"
+      data-qa="big"
+    >
+      <label for="search__bar--big" class="search__labl search__labl--big">
+        <div class="search__icon search__icon--big"></div>
+      </label>
+
+      <input type="search"
+        name="search__bar"
+        id="search__bar--big"
+        class="search__bar search__bar--big"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+     >
```

Comment:
Let's fix it everywhere
```suggestion
      <input
        type="search"
        name="search__bar"
        id="search__bar--big"
        class="search__bar search__bar--big"
        placeholder="Try “Los Angeles”"
        data-qa="keypress"
     >
```